### ISSUE TITLE
Feature request: no option to close/ignore a lint message in the IDE

### ISSUE THREAD

**Issue by ellet0** [nbadal/ktlint-intellij-plugin#522]
## Observed Behavior
The Intellij IDEA [Ktlint](https://plugins.jetbrains.com/plugin/15057-ktlint) plugin shows the message 
`Ktlint found <number> lint violations of which <number> can be autocorrected>`

![image](https://github.com/pinterest/ktlint/assets/73608287/d3b47f82-5a6f-4713-bc20-1263ea5e5194)

## Expected Behavior

It would be useful to show the message only once or add an option to close the message as showing it multiple times can cover the whole editor or even the screen, and it keeps going infinitely

## Steps to Reproduce

This issue report is not very useful because I'm not sure how to reproduce it (and probably will never be),
you can close the issue as unplanned if you would like to

I can provide any information you want

 I mostly focus on the code, not the message, but there is one way that might work which is to cause too many lint violations in the project (around 25 or more), and then once the message shows keep working and ignore it and it might duplicate more (after several hours maybe)

## Your Environment

The Ktlint mode is not set to any option in the project, but I think it did happen on another project where I actually have Ktlint enabled

![image](https://github.com/pinterest/ktlint/assets/73608287/92da0fa4-079e-450f-bfc5-dd848ff77569)

![image](https://github.com/pinterest/ktlint/assets/73608287/337ae09d-cb0c-48f1-bd6e-b3af67426697)

IntelliJ IDEA info:

```

IntelliJ IDEA 2024.1.1 (Community Edition)
Build #IC-241.15989.150, built on April 29, 2024
Runtime version: 17.0.10+1-b1207.14 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.2.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 10
Metal Rendering is ON
Registry:
  ide.experimental.ui=true
  terminal.new.ui=true
Non-Bundled Plugins:
  com.jetbrains.space (241.15989.9)
  com.nbadal.ktlint (0.22.0)
  com.varabyte.kobweb (0.1.0)
Kotlin: 241.15989.150-IJ


```

* Version of ktlint used: IDE plugin version is 0.22.0
* Relevant parts of the `.editorconfig`: I didn't use it
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): for this project, I didn't enable Ktlint Gradle plugin or use it, the message kept showing in `.kt` and `.kts` files, which is okay, but it would be nice to have an option to close/ignore it or fix that bug cause the message to be duplicated multiple times
* Version of Gradle used (if applicable): 8.7
* Operating System and version: macOS m1 (14.2.1 (23C71))

Thank you.

**Comment by paul-dingemans** [comment]
I can not reproduce with 300+ violations in a single file.

![Screenshot 2024-05-16 at 18 47 46](https://github.com/nbadal/ktlint-intellij-plugin/assets/5195292/27984ba0-6f44-4815-87d1-b1790d7c047e)

* Does the problem disappear after restart of IDEA?
* What version of the IDEA are you using?

You can get rid of the green message bar by going to the ktlint settings and make a choice for one the modes. This needs to be done for each project.

**Comment by ellet0** [comment]
> I can not reproduce with 300+ violations in a single file.
> 
> ![Screenshot 2024-05-16 at 18 47 46](https://github.com/nbadal/ktlint-intellij-plugin/assets/5195292/27984ba0-6f44-4815-87d1-b1790d7c047e)
> 
> * Does the problem disappear after restart of IDEA?
> * What version of the IDEA are you using?
> 
> You can get rid of the green message bar by going to the ktlint settings and make a choice for one the modes. This needs to be done for each project.
> 
> 
> 

Yes, it disappears after restarting the project. The version of the IDE is:
IntelliJ IDEA 2024.1.1 (Community Edition)
Build #IC-241.15989.150, built on April 29, 2024

**Comment by paul-dingemans** [comment]
>> You can get rid of the green message bar by going to the ktlint settings and make a choice for one the modes. This needs to be done for each project.
>
> I might not be the only one working on the project, doing this change will update .idea/ktlint-plugin.xml and other developers might not want this or don't have this issue and this file is not ignored in .gitignore by default.

You might consider to discuss with other developers on the project to make use of ktlint mandatory. If that is not acceptible, the way to go would be to add the `.idea/ktlint-plugin.xml` to the `.gitignore` so that every developer can choose indiviually which configuration to use. Note that having this configuration file will not affect users that do not have the `ktlint-intellij-plugin` installed.

**Comment by ellet0** [comment]
The project is open source, and contributions come from different developers. Regarding the issue, I have no idea how to reproduce it, which is why this is a feature request.

I understand that you don't have plans on implementing this feature of closing the message in IDE.

Thank you once again.
